roles/ceph-osd/tasks/pre_requisite.yml

```diff
@@ -11,6 +11,7 @@
   package:
     name: parted
     state: present
+  when: ansible_os_family != 'ClearLinux'
```

Comment:
ClearLinux does not manage packages but "bundles" instead.

The parted package is included already in the storage-cluster bundle